<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(89, 122, 170, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(89, 122, 170, 255), stop:1 rgba(255, 255, 255, 255));}</string>
      </property>
      <widget class="QPushButton" name="btnProfileSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/settings.png</normaloff>images/settings.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnExitSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>400</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(74, 0, 0);
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/exit.png</normaloff>images/exit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStatsSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>79</y>
         <width>100</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/analytics.png</normaloff>images/analytics.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRatingSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/rating.png</normaloff>images/rating.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnResourcesSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>240</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/milk-box.png</normaloff>images/milk-box.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRefillSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/coffee-filter.png</normaloff>images/coffee-filter.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="loginPage" native="true">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>699</width>
         <height>480</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(89, 122, 170, 255), stop:1 rgba(255, 255, 255, 255));}</string>
       </property>
       <widget class="QLabel" name="lblError">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>249</y>
          <width>391</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;; color:red;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="LogoProfile">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>images/settings.png</pixmap>
        </property>
       </widget>
       <widget class="QLineEdit" name="leUsername">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>210</y>
          <width>301</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0,0,0,0);
border: 1px solid black;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="lblUsername">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Username:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="txtWelcome">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>631</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Please log-in first or scan your chip!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>-2</number>
        </property>
       </widget>
       <widget class="QPushButton" name="btnLogin">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>309</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:15px;
background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblPassword">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>180</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblRfidLogo">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>90</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>images/rfid.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lePassword">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>210</y>
          <width>291</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0,0,0,0);
border: 1px solid black;
</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionLogin">
   <property name="icon">
    <iconset>
     <normaloff>images/log-in.png</normaloff>images/log-in.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionSign_up">
   <property name="icon">
    <iconset>
     <normaloff>images/sign-up.png</normaloff>images/sign-up.png</iconset>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </action>
  <action name="actionStats">
   <property name="icon">
    <iconset>
     <normaloff>images/analytics.png</normaloff>images/analytics.png</iconset>
   </property>
   <property name="text">
    <string>Stats</string>
   </property>
  </action>
  <action name="actionResources">
   <property name="icon">
    <iconset>
     <normaloff>images/milk-box.png</normaloff>images/milk-box.png</iconset>
   </property>
   <property name="text">
    <string>Resources</string>
   </property>
  </action>
  <action name="actionProfile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/settings.png</normaloff>images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionRefill">
   <property name="icon">
    <iconset>
     <normaloff>images/coffee-filter.png</normaloff>images/coffee-filter.png</iconset>
   </property>
   <property name="text">
    <string>Refill</string>
   </property>
  </action>
  <action name="actionRating">
   <property name="icon">
    <iconset>
     <normaloff>images/rating.png</normaloff>images/rating.png</iconset>
   </property>
   <property name="text">
    <string>Rating</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
